src/axios.ts

```diff
@@ -86,52 +89,54 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
     });
   };
 
-  Proto.searchUsers = function(query: string): AxiosPromise<UserSpec[]> {
-    return this.get('/user/search', {
+  Proto.searchUsers = function(username: string): AxiosPromise<UserSpec[]> {
+    return this.get('/users/search', {
       params: {
-        query,
+        username,
       },
     });
   };
 
-  Proto.tables = function(workspace: string, options: TablesOptionsSpec = {}): AxiosPromise<string[]> {
+  Proto.tables = function(workspace: string, options: TablesOptionsSpec = {}): AxiosPromise<Paginated<Table>> {
     return this.get(`workspaces/${workspace}/tables`, {
       params: options,
     });
   };
 
-  Proto.table = function(workspace: string, table: string, options: OffsetLimitSpec = {}): AxiosPromise<RowsSpec> {
-    return this.get(`workspaces/${workspace}/tables/${table}`, {
+  Proto.table = function(workspace: string, table: string, options: OffsetLimitSpec = {}): AxiosPromise<Paginated<TableRow>> {
+    return this.get(`workspaces/${workspace}/tables/${table}/rows`, {
       params: options,
     });
   };
 
-  Proto.graphs = function(workspace: string): AxiosPromise<string[]> {
-    return this.get(`workspaces/${workspace}/graphs`);
+  Proto.graphs = function(workspace: string): AxiosPromise<Paginated<Graph>> {
+    return this.get(`workspaces/${workspace}/networks`);
   };
 
   Proto.graph = function(workspace: string, graph: string): AxiosPromise<GraphSpec> {
-    return this.get(`workspaces/${workspace}/graphs/${graph}`);
+    return this.get(`workspaces/${workspace}/networks/${graph}`);
   };
 
-  Proto.nodes = function(workspace: string, graph: string, options: OffsetLimitSpec = {}): AxiosPromise<NodesSpec> {
-    return this.get(`workspaces/${workspace}/graphs/${graph}/nodes`, {
+  Proto.nodes = function(workspace: string, graph: string, options: OffsetLimitSpec = {}): AxiosPromise<Paginated<TableRow>> {
+    return this.get(`workspaces/${workspace}/networks/${graph}/nodes`, {
       params: options,
     });
   };
 
   Proto.attributes = function(workspace: string, graph: string, nodeId: string): AxiosPromise<{}> {
-    return this.get(`workspaces/${workspace}/graphs/${graph}/nodes/${nodeId}/attributes`);
+    return this.get(`workspaces/${workspace}/networks/${graph}/nodes/${nodeId}/attributes`);
   };
 
-  Proto.edges = function(workspace: string, graph: string, nodeId: string, options: EdgesOptionsSpec = {}): AxiosPromise<EdgesSpec> {
-    return this.get(`workspaces/${workspace}/graphs/${graph}/nodes/${nodeId}/edges`, {
+  Proto.edges = function(workspace: string, graph: string, nodeId: string, options: EdgesOptionsSpec = {}): AxiosPromise<Paginated<EdgesSpec>> {
```

Comment:
`nodeId` isn't used here anymore, so it should be removed

Reply:
See https://github.com/multinet-app/multinetjs/pull/42/commits/439d2750bc9856cc074495da6db83f328736e439#diff-69f7f6dfc4bdae4e37eddc7d92ffdf1ed2d3fdc99a9c55068c5b4790a157ee79L130-R125

Reply:
I initially forgot to remove other instances of the parameter, so see also 1493525